short-read-mngs/idseq-dag/idseq_dag/steps/generate_coverage_stats.py

```diff
@@ -176,6 +176,10 @@ def compute_slice(slice_idx):
                 }
             )
         )
+        # if all intermediate json files are empty, close collated json file
+        json_sizes = sum([os.path.getsize(tfn) for tfn in output_json_filenames[:-1]])
+        if json_sizes == 0:
+            ojson.write("}")
```

Comment:
This seems like a typo, should it be `json` not `ojson`?

Reply:
sorry, forgot to open a json file. it's meant to match the opening statement.